app/src/main/java/com/gh4a/fragment/UserFragment.java

```diff
@@ -85,8 +91,8 @@ protected void onResultReady(User result) {
         @Override
         protected Loader<LoaderResult<Collection<Repository>>> onCreateLoader() {
             Map<String, String> filterData = new HashMap<>();
-            filterData.put("sort", "pushed");
-            filterData.put("affiliation", "owner,collaborator");
+            filterData.put(EXTRA_SORT, PUSHED);
+            filterData.put(EXTRA_AFFILIATION, "owner,collaborator");
```

Comment:
Those aren't extras, but filter fields. Please move those to RepositoryService (see IssueService for example on variable naming).